coalahtml/tree.py

```diff
@@ -0,0 +1,81 @@
+from collections import OrderedDict
+from queue import Queue
+import os
+
+
+class Node:
+    """
+    Node class that defines an individual node.
+    """
+
+    def __init__(self, path):
+        self.child = {}
+        self.parent = None
+        self.path = path
+        self.abspath = ""
+        self._id = id(self)
```

Comment:
Um I actually thought maybe I could use the `_id` attribute to identify the nodes if needed in future.